gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -190,8 +190,8 @@ private void onToolbarLayout() {
   private void setUpBottomSheetBehavior() {
     bottomSheetBehavior = BottomSheetBehavior.from(bottomSheetScrollView);
     bottomSheetBehavior.setHideable(true);
-    bottomSheetBehavior.setState(BottomSheetBehavior.STATE_HIDDEN);
     bottomSheetBehavior.setBottomSheetCallback(new BottomSheetCallback());
+    bottomSheetBehavior.setState(BottomSheetBehavior.STATE_COLLAPSED);
```

Comment:
It seems the problem is that this method is getting called via `onViewCreated()` on back because the view is getting recreated. The root cause appears to be that the BottomSheetBehavior's previous state is not being preserved when the view is recreated.

I just found this in the [BottomSheetBehavior#onInstanceState docs](https://developer.android.com/reference/android/support/design/widget/BottomSheetBehavior#onsaveinstancestate):
> Behavior state is only saved when both the parent CoordinatorLayout and a view using this behavior have valid IDs set.

If you assign a valid id to the `CoordinatorLayout` in `home_screen_frag.xml`, and only call `setUpBottomSheetBehavior()` when `savedInstanceState == null`, that should fix the root cause. HTH!